sys/parsoid.js

```diff
@@ -246,23 +289,41 @@ class ParsoidService {
         return hyper.request(newReq);
     }
 
-    saveParsoidResult(hyper, req, format, tid, parsoidResp) {
+    saveParsoidResultToLatest(hyper, req, tid, parsoidResp) {
+        const rp = req.params;
+        return hyper.put({
+            uri: this.getNGBucketURI(rp, 'all', tid),
+            body: {
+                html: parsoidResp.body.html,
+                'data-parsoid': parsoidResp.body['data-parsoid'],
+                'section-offsets': {
+                    headers: {
+                        'content-type': 'application/json'
+                    },
+                    body: parsoidResp.body['data-parsoid'].body.sectionOffsets
```

Comment:
If we need to set the content for data-parsoid, then we need to set it here as well. If we don't, then we don't need it anywhere.

Reply:
Uh, I placed the wrong comment on the wrong line, sorry about that. My point is that here in the request we explicitly set the `content-type` header for `section-offsets`, but not for `data-parsoid`, so the way I see it we should either set the header for both components or for neither of them, shouldn't we?

Reply:
@d00rman no, not really. For `data-parsoid` we set `parsoidResp.body['data-parsoid']` and for the `pagebundle` response from parsoid that property contains both `headers` and `body`. For `section-offsets` we store a subject of the `data-parsoid` body, so we need to specify the content type to something. We could either reuse the same headers as `data-parsoid` has or just state it's application/json. I'm indifferent between both approaches though.

Reply:
Ah indeed @Pchelolo !